validator/remote/src/main/java/tech/pegasys/teku/validator/remote/apiclient/OkHttpValidatorRestApiClient.java

```diff
@@ -157,12 +157,19 @@ public OkHttpValidatorRestApiClient(final HttpUrl baseEndpoint, final OkHttpClie
 
   @Override
   public Optional<BeaconBlock> createUnsignedBlock(
-      final UInt64 slot, final BLSSignature randaoReveal, final Optional<Bytes32> graffiti) {
+      final UInt64 slot,
+      final BLSSignature randaoReveal,
+      final Optional<Bytes32> graffiti,
+      final boolean blinded) {
     final Map<String, String> pathParams = Map.of("slot", slot.toString());
     final Map<String, String> queryParams = new HashMap<>();
     queryParams.put("randao_reveal", encodeQueryParam(randaoReveal));
     graffiti.ifPresent(bytes32 -> queryParams.put("graffiti", encodeQueryParam(bytes32)));
 
+    if (blinded) {
+      throw new UnsupportedOperationException("blinded blocks not yet implemented");
```

Comment:
Probably worth logging a ticket to follow up on this so it doesn't get forgotten.

Reply:
yes, currently it's a checkbox in #5102. I'll transform them in issues.

Reply:
If you don't mind I'll leave the main issue to avoid noise. I'll transform them to tickets when I start there If will make sense.